### ISSUE TITLE
Getting "Network Protocol Error" if mod is in the server 

### ISSUE THREAD

**Issue by chocotoneg** [Globox1997/RpgDifficulty#79]
### Minecraft Version

1.21.x

### Mod Loader Version

0.16.0

### Mod Version

1.3.13

### Describe the Issue

Whenever i tried joining the server i'd get a "Network Protocol Error", after some binary search it seemed like RpgDifficulty was causing the error

### latest.log

Errors below [Line 959](https://gist.github.com/chocotoneg/5cf147e4c16f261e972ffc1eb84b8176) (Note: it's not REI, tested without it cause i thought it was a bit sus by the log and previous ones)
No errors in console, only client.

### Do you use any other mods except the required ones?

Yes

**Comment by Globox1997** [comment]
So without RPGDifficulty, everything works fine?

**Comment by chocotoneg** [comment]
yes, when i removed your mod it worked just fine

**Comment by elhertz** [comment]
same problem here
edit: it kicks me sometimes with the same error and when i finally manage to join the server will freeze for some reason and i have to kill the console

**Comment by chocotoneg** [comment]
still having issues in 1.21.1 Fabric 0.16.5.

**Comment by Globox1997** [comment]
How can I reproduce this bug?
The log you sent here is about a bundle item sync issue.

**Comment by chocotoneg** [comment]
i guess the only way of reproducing this bug is using the same modpack i am.
Also, thats what i find weird because i was getting kicked with my inventory empty.

**Comment by House-Kitty-Mew** [comment]
You wouldn't happen to have Travelerz in you'r modpack? i was getting the same issue until i removed that, turns out its compatibility is a version behind the mod that runs off the mod name plate that runs off rpgdiffaculty, its a chain problem where travelerz isn't updated to handle the new way of protocol implemented in 1.20.5

> i guess the only way of reproducing this bug is using the same modpack i am. Also, thats what i find weird because i was getting kicked with my inventory empty.

Ref:
I’m running Travelerz on Minecraft 1.21.1 with RPGDifficulty and it crashes immediately on server start with:

java.lang.NoSuchMethodError: 

  'void net.rpgdifficulty.api.MobStrengthener.changeAttributes(net.minecraft.class_1308, net.minecraft.class_1937)'
    at net.travelerz.network.TravelerServerPacket.lambda$init$0(TravelerServerPacket.java:26)

It looks like Travelerz is calling MobStrengthener.changeAttributes(...), but that method no longer exists in the RPGDifficulty API I’ve got installed. Probably it was renamed, removed, or its signature changed.

**Comment by chocotoneg** [comment]
no, this was almost a year ago too- the modpack that had it in died. But i checked the logs and Travelerz wasn't present.

**Comment by Globox1997** [comment]
This got fixed with the latest version of TravelerZ.
